elastica/joint.py

```diff
@@ -222,51 +228,68 @@ def __init__(self, k, nu, kt):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float = 0.
+            Rotational damping coefficient of the joint.
+        use_static_rotation: bool
+            If True, the initial rotation offset between the two systems is recorded
+            and enforced throughout the entire simulation. (default=True)
```

Comment:
```suggestion
        is_rest_joint_state: bool
            If True, the initial rotation angle between the two systems is recorded
            and enforced throughout the entire simulation. (default=True)
```

Reply:
Resolved in https://github.com/GazzolaLab/PyElastica/pull/135/commits/9a5cf1cb90ebd7fb6a215d799eb27c5d0bd1c5f0